src/Microsoft.AspNet.Identity.EntityFramework/UserStore.cs

```diff
@@ -940,7 +961,7 @@ public virtual Task<bool> GetTwoFactorEnabledAsync(TUser user, CancellationToken
                         && userclaims.ClaimType == claim.Type
                         select user;
 
-            return (IList<TUser>)await query.ToListAsync(cancellationToken);
+            return await query.ToListAsync(cancellationToken);
```

Comment:
yeah I just reflexively clean up grey'd out VS code that is redundant.  Compiler is perfectly happy without the cast